project.clj

```diff
@@ -1,4 +1,6 @@
-(defproject clj-statsd "0.3.11"
+(defproject shmish111/clj-statsd "0.4.0"
```

Comment:
Please keep the artifact description as previously. I'm fine with bumping to 0.4.0